src/scripts/main.js

```diff
@@ -1 +1,50 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('context menu', () => {
+    resolve('Second promise was resolved');
+  });
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let leftClicked = false;
+  let rightClicked = false;
+
+  document.addEventListener('click', () => {
+    leftClicked = true;
+    rightClicked === true && resolve('Third promise was resolved');
```

Comment:
better to rewrite to:
```suggestion
    if (rightClicked) resolve('Third promise was resolved');
```